src/components/search/Search.jsx

```diff
@@ -1,9 +1,39 @@
-import React from 'react'
+import React, { Component } from "react";
+import "./_Search.scss";
+import searchIcon from "assets/search-solid.png";
 
-const Search = () => {
-    return <div>
-        <input type="text" placeholder="Search here..."/>
-    </div>
+class Search extends Component {
+  constructor() {
+    super();
```

Comment:
You should pass the props to the constructor and to the super call too

Reply:
Attended